src/app/policies/views/PolicyTypeListView.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <DataSource
+    v-slot="{ data: me }: MeSource"
+    src="/me"
+  >
+    <RouteView
+      v-if="me"
+      v-slot="{ route, t }"
+      name="policy-list-view"
+      :params="{
+        page: 1,
+        size: me.pageSize,
+        mesh: '',
+        policyPath: '',
+        policy: '',
+      }"
+    >
+      <AppView>
+        <template #title>
+          <h2>
+            <RouteTitle
+              :title="t('policies.routes.items.title')"
+            />
+          </h2>
+        </template>
+        <DataSource
+          v-slot="{ data: meshInsight }: MeshInsightSource"
+          :src="`/mesh-insights/${route.params.mesh}`"
+        >
+          <DataSource
+            v-slot="{ data }: PolicyTypeCollectionSource"
+            :src="`/*/policy-types`"
+          >
+            <div
+              class="policy-list-content"
+            >
+              <KCard
+                class="policy-type-list"
+                data-testid="policy-type-list"
+              >
+                <!-- block on policy types but not meshInsight -->
+                <DataLoader
+                  :data="[data]"
+                >
```

Comment:
**Question**: Is there a better way to do this? Using a DataLoader in a strictly not data loading capacity is extremely odd/unintuitive.

Reply:
Not sure if I follow. This is using DataLoader in a data loading capacity. It is blocking on policyTypes loading and showing a loading spinner inside the KCard until they are loaded.

---

Separately, one thing I noticed that I think we are missing here is the error state / `:errors-[error]`. I'll add that in shortly, ty!

Reply:
I added a commit to pass through the error to the DataLoader here. Let me know if we can resolve this one

Reply:
I'm gonna resolve this one, as I think its fine now, let me know if not and we can reopen.